pyproject.toml

```diff
@@ -19,17 +23,14 @@ dev = [
     "pytest-cov",
 ]
 docs = [
-    # Sphinx extensions
     "autodoc",
     "mathjax",
-    # Third-party extensions
     "myst-parser",
 ]
 
-[tools.setuptools]
-packages = [
-    { include = "pbeis" },
-]
+[tool.setuptools.packages.find]
+where = ["."]
+include = ["pbeis"]
```

Comment:
should we also rename this to pybammeis? so it's not confusing for users? same for examples etc.?

also, the installation instructions can be updated for a user install

Reply:
I can rename that as well if you want, but having different names is possible. The way I have it set up now is that you install with
```
pip install pybammeis
```
but still import with
```
import pbeis
```

If you want them to be same that is fine though.

Reply:
yeah I think it might be confusing if they are different